packages/apollo/src/rest/ChannelApi.js

```diff
@@ -556,12 +556,12 @@ class ChannelApi {
 	static getOrdererAddresses(config) {
 		let addresses = [];
 		const orderer_grp = _.get(config, 'channel_group.groups_map.Orderer.groups_map');
```

Comment:
Can we move this to the addresses.lengh loop so that we do it only when needed?

Reply:
updated